windows/shared/logging/PlogTraceLogging/CMakeLists.txt

```
@@ -0,0 +1,36 @@
+
+add_library(plog-trace-logging STATIC "")
+
+set(PLOGTRACELOGGING_DIR_PUBLIC_INCLUDE ${CMAKE_CURRENT_LIST_DIR}/include)
+set(PLOGTRACELOGGING_DIR_PUBLIC_INCLUDE_PREFIX ${PLOGTRACELOGGING_DIR_PUBLIC_INCLUDE}/PlogTraceLogging)
+
+target_sources(plog-trace-logging
+    PRIVATE
+        ${CMAKE_CURRENT_LIST_DIR}/PlogTraceLogging.cxx
+    PUBLIC
+        ${PLOGTRACELOGGING_DIR_PUBLIC_INCLUDE_PREFIX}/PlogTraceLogging.hxx
+)
+
+target_include_directories(plog-trace-logging
+    PUBLIC
+        ${PLOGTRACELOGGING_DIR_PUBLIC_INCLUDE}
+)
+
+target_link_libraries(plog-trace-logging
+PUBLIC
+    plog::plog

```

Looks like tabbing is off here.
```suggestion
    PUBLIC
        plog::plog

```